presto-main/src/test/java/com/facebook/presto/operator/aggregation/differentialentropy/TestReservoirAggregation.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.differentialentropy;
+
+import com.facebook.presto.operator.aggregation.AbstractTestAggregationFunction;
+import com.facebook.presto.operator.aggregation.InternalAggregationFunction;
+
+import static com.facebook.presto.operator.aggregation.differentialentropy.EntropyUtils.calculateEntropyFromSamples;
+import static org.testng.Assert.assertTrue;
+
+public abstract class TestReservoirAggregation
+        extends AbstractTestAggregationFunction
+{
+    protected static final Integer SIZE = 500;
```

Comment:
- Is this maximum number of samples? If so, rename to `MAX_SAMPLES`.
- change type to `int`

`protected static final int MAX_SAMPLES = 500;`